src/Sylius/Bundle/ImportExportBundle/Behat/ImportExportContext.php

```diff
@@ -0,0 +1,663 @@
+    public function iShouldFindJobForThisProfileInDatabase($numberOfJobs, $jobStatus, $profileType)
+    {
+        $repository = $this->getRepository($profileType.'_job');
+        $numberOfJobsInDatabase = count($repository->findBy(array('status' => $jobStatus)));
+
+        if ((int) $numberOfJobs !== $numberOfJobsInDatabase) {
+            $executionMessage = sprintf('Expected was %d jobs with status "%s" in database, but %d was found',
+                $numberOfJobs,
+                $jobStatus,
+                $numberOfJobsInDatabase
+            );
+
+            throw new \Exception($executionMessage);
+        }
+    }
+
+    /**
+     * @Then I should see  :message in error message
+     */
+    public function iShouldFindInErrorMessage($message)
+    {
+        if (false === strpos($this->processOutput, $message)) {
+            $executionMessage = sprintf('"%s" was not found in error message "%s"',
+                $message,
+                $this->processOutput
+            );
+
+            throw new \Exception($executionMessage);
+        }
+    }
+
+    /**
+     * @Given there are following import jobs set:
+     */
+    public function thereAreFollowingImportJobsSet(TableNode $table)
+    {
+        $manager = $this->getEntityManager();
+
+        foreach ($table->getHash() as $data) {
+            $this->thereIsImportJob($data['status'], $data['start_time'], $data["end_time"], $data['created_at'], $data['updated_at'], $data["import_profile"], false);
+        }
+
+        $manager->flush();
+    }
+
+    /**
+     * @Given I am on the import jobs index page for profile with code :code
+     * @And I am on the import jobs index page for profile with code :code
+     */
+    public function iAmOnTheImportJobsIndexPageForProfileWithCode($code)
+    {
+        $importProfile = $this->findOneBy('import_profile', array('code' => $code));
+
+        $this->getSession()->visit($this->generatePageUrl('sylius_backend_import_job_index', array('profileId' => $importProfile->getId())));
+    }
+
+    /**
+     * @Given I press :button near :value import job
+     * @Given I click :button near :value import job
+     */
+    public function iPressNearImportJob($button, $value)
+    {
+        $tr = $this->assertSession()->elementExists('css', sprintf('table tbody tr:nth-child(%s)', $value));
+
+        $locator = sprintf('button:contains("%s")', $button);
+
+        if ($tr->has('css', $locator)) {
+            $tr->find('css', $locator)->press();
+
+            return;
+        }
+
+        $tr->clickLink($button);
+    }
+
+    /**
+     * @Then I should be on the import jobs index page for profile with code :code
+     */
+    public function iShouldBeOnTheImportJobsIndexPageForProfileWithCode($code)
+    {
+        $importProfile = $this->findOneBy('import_profile', array('code' => $code));
+
+        $this->getSession()->visit($this->generatePageUrl('sylius_backend_import_job_index', array('profileId' => $importProfile->getId())));
```

Comment:
If step is "I should be...", shouldn't you assert that you are on specific page, rather than use ``visit`` function?